### PR TITLE
Update the list of GraphQL clients in the docs to include Postman

### PR DESCRIPTION
**Pull request by meenakshi-dhanani**
Screenshot 2023-11-10 at 3 52 17 PM

The screenshot above displays the addition of Postman to list of clients such as GraphiQL and Insomnia mentioned in the docs.

### CODE REVIEW

**Comment by benjie** [comment]
Reopening in the hopes it kickstarts Gatsby into retrying

**Comment by Urigo** [comment]
image

It should be after Insomnia

It breaks the md syntax

**Comment by meenakshi-dhanani** [comment]
> image
> 
> It should be after Insomnia
> 
> It breaks the md syntax



> image
> 
> It should be after Insomnia
> 
> It breaks the md syntax

Do you mean this? 
such as [GraphiQL](https://github.com/graphql/graphiql), [Insomnia](https://github.com/getinsomnia/insomnia), and [Postman](https://www.postman.com/product/graphql-client/).

I hoped the prettier check would catch any missing or wrongly formatted markdown.

**Comment by benjie** [comment]
I don’t see the mistake @Urigo? Also can you grant me access to see the error logs, please?

**Comment by meenakshi-dhanani** [comment]
@Urigo if you can confirm [GraphiQL](https://github.com/graphql/graphiql), [Insomnia](https://github.com/getinsomnia/insomnia), and [Postman](https://www.postman.com/product/graphql-client/). - I can modify and raise the PR.
But yes, I haven't been able to reproduce that error locally, my local build has this failing 
Screenshot 2023-11-15 at 5 42 22 PM

File: src/content/graphql-js/Tutorial-GraphQLClients.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ You should see the output returned as JSON:
 {"data":{"hello":"Hello world!"}}
 ```
 
-If you prefer to use a graphical user interface to send a test query, you can use clients such as [GraphiQL](https://github.com/graphql/graphiql) and [Insomnia](https://github.com/getinsomnia/insomnia).
+If you prefer to use a graphical user interface to send a test query, you can use clients such as [GraphiQL](https://github.com/graphql/graphiql), [Insomnia](https://github.com/getinsomnia/insomnia) and [Postman](https://www.postman.com/product/graphql-client/).
```
Comment (YassinEldeeb) [inline src/content/graphql-js/Tutorial-GraphQLClients.md]:
```suggestion
If you prefer to use a graphical user interface to send a test query, you can use clients such as [GraphiQL](https://github.com/graphql/graphiql), [Insomnia](https://github.com/getinsomnia/insomnia), and [Postman](https://www.postman.com/product/graphql-client/).
```

**Comment by Urigo** [comment]
deployed and in master, sorry it took so long @meenakshi-dhanani , we are now using Vercel instead of Gatsby Cloud

**Pull request closed without merging**